### PR TITLE
Update Hungarian translation to 1.16.0

### PR DESCRIPTION
**Pull request by smileyhead**

### CODE REVIEW

File: resources/hu/cemu.po [hunk 1]
```diff
@@ -545,77 +678,81 @@ msgstr ""
...
-#: GeneralSettings2.cpp:124
-msgid "Saves a screenshot directly to a file"
-msgstr "Egy fájlba menti a képernyőmentést"
+#: GeneralSettings2.cpp
+msgid ""
+"Pressing the screenshot key (F12) will save a screenshot directly to the "
+"screenshots folder"
+msgstr ""
+"A képernyőmentés gomb (F12) lenyomása egy képernyőmentést helyez a "
+"képernyőmentések mappába"
```
Comment (Fs00) [inline resources/hu/cemu.po]: The meaning of the original text is a bit different: "screenshots folder" refers to a folder named "screenshots", not to a generic folder containing screenshots

File: resources/hu/cemu.po [hunk 2]
```diff
@@ -932,44 +1081,51 @@ msgstr ""
...
+msgstr "Monó"
+
+#: GeneralSettings2.cpp
 msgid "Stereo"
 msgstr "Sztereó"
 
-#: GeneralSettings2.cpp:428
+#: GeneralSettings2.cpp
+msgid "Surround"
+msgstr "Surround Sound"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Why not just "Surround"? 🤔

Comment (smileyhead) [reply resources/hu/cemu.po]: Hmm, this one isn't a recent string, so I don't really remember my reasoning behind it anymore, but I'll translate it properly now.

**Comment by Fs00** [comment]
Thanks!

**Pull request merged**